scripts/releaseChangelog.js

```diff
@@ -143,7 +144,13 @@ async function main(argv) {
       case 'DataGridPremium':
       case 'l10n':
       case '118n':
-        changeCommits.push(commitItem);
+        dataGridCommits.push(commitItem);
+        break;
+      case 'DatePicker':
+      case 'TimePicker':
+      case 'DateRangePicker':
+      case 'pickers':
+        pickersCommits.push(commitItem);
```

Comment:
I only listed the main public components because the list is huge. With more components (e.g. charts) we do need to think in a more scalable solution.

Reply:
Yup, and even now, `[i18n]` could be both for pickers or grid changes. 🤔

Reply:
Maybe [l10n] should have their own section like for [core] and [docs]